root/release_group/ReleaseGroupIndex.js

```diff
@@ -0,0 +1,213 @@
+              ? <th />
+              : null
+          )}
+        </CatalystContext.Consumer>
+        <CatalystContext.Consumer>
+          {($c: CatalystContextT) => (
+            <th colSpan={$c.session && $c.session.tport ? 9 : 8}>
+              {status && status.name
+                ? lp_attributes(status.name, 'release_status')
+                : l('(unknown)')}
+            </th>
+          )}
+        </CatalystContext.Consumer>
+      </tr>
+      {releaseStatusGroup.map((release, index) => (
+        <tr className={loopParity(index)} key={release.id}>
+          <CatalystContext.Consumer>
+            {($c: CatalystContextT) => (
+              $c.user_exists
+                ? (
+                  <td>
+                    <input
+                      name="add-to-merge"
+                      type="checkbox"
+                      value={release.id}
+                    />
+                  </td>
+                ) : null
+            )}
+          </CatalystContext.Consumer>
+          <td>
+            <EntityLink entity={release} />
+          </td>
+          <td>{release.combined_format_name || l('[missing media]')}</td>
+          <td>{release.combined_track_count || '-'}</td>
+          <td>
+            <ReleaseDates events={release.events} />
+          </td>
+          <td>
+            <ReleaseCountries events={release.events} />
+          </td>
+          <td>
+            <ReleaseLabelList labels={release.labels} />
+          </td>
+          <td>
+            <ReleaseCatnoList labels={release.labels} />
+          </td>
+          <td className="barcode-cell">{formatBarcode(release.barcode)}</td>
+          <CatalystContext.Consumer>
+            {($c: CatalystContextT) => (
+              $c.session && $c.session.tport
+                ? <td><TaggerIcon entity={release} /></td>
+                : null
+            )}
+          </CatalystContext.Consumer>
+        </tr>
+      ))}
+    </>
+  );
+}
+
+const ReleaseGroupIndex = ({
+  $c,
+  eligibleForCleanup,
+  pager,
+  mostPopularReview,
+  mostRecentReview,
+  numberOfRevisions,
+  releaseGroup,
+  releases,
+  wikipediaExtract,
+}: Props) => (
+  <ReleaseGroupLayout entity={releaseGroup} page="index">
+    {eligibleForCleanup ? (
+      <p className="cleanup">
+        {l(
+          `This release group has no relationships or releases associated,
+           and will be removed automatically in the next few days. If this
+           is not intended, please add more data to this release group.`,
+        )}
+      </p>
+    ) : null
+    }
+    <Annotation
+      annotation={releaseGroup.latest_annotation}
+      collapse
+      entity={releaseGroup}
+      numberOfRevisions={numberOfRevisions}
+    />
+    <WikipediaExtract
+      cachedWikipediaExtract={wikipediaExtract || null}
+      entity={releaseGroup}
+    />
+    {releases.length ? (
+      <>
+        {releaseGroup.typeName ? <h2>{l(releaseGroup.typeName)}</h2> : null}
```

Comment:
```suggestion
        {releaseGroup.typeName ? (
          <h2>{lp_attributes(releaseGroup.typeName, 'release_group_primary_type')}</h2>
        ) : null}
```

Reply:
Actually that wouldn't work, but neither would my version. Changed to releaseGroupType(releaseGroup) since this has to support type combos.

Reply:
Thanks, I forgot l_type_name did that